### PR TITLE
LSR-1004 iOS: PortletDisplayScreenlet

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

File: ios/Framework/Core/Portlet/html/index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<html>
+	<head>
+		<script>
+			function post() {
+				var method = "post";
```
Comment (nhpatt) [inline ios/Framework/Core/Portlet/html/index.html]: 'post'

File: ios/Framework/Core/Portlet/html/index.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+				var form = document.createElement("form");
+				form.setAttribute("method", method);
+				form.setAttribute("action", "/c/portal/login?redirect=<portletUrl>");
+
+				let params = {
```
Comment (nhpatt) [inline ios/Framework/Core/Portlet/html/index.html]: you can use let?

File: ios/Framework/Core/Portlet/html/index.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+					login: "<login>",
+					password: "<password>"
+				};
+				
+				for(var key in params) {
```
Comment (nhpatt) [inline ios/Framework/Core/Portlet/html/index.html]: for (

File: ios/Framework/Core/Portlet/js/Screens.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
+const log = console.log;
```
Comment (nhpatt) [inline ios/Framework/Core/Portlet/js/Screens.js]: 2 or 4? can't be both

File: ios/Framework/Core/Portlet/js/Screens.js [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+  addScreensScript: function(screensScript) {
+    this.screensScripts_.push(screensScript);
+  },
+
+  reloadScripts: function() {
+    this.screensScripts_.forEach(fn => fn());
+  },
+
+  isAndroid: function() {
+    if (navigator.userAgent.indexOf("Android") === -1) {
```
Comment (nhpatt) [inline ios/Framework/Core/Portlet/js/Screens.js]: return !=

File: ios/Samples/Showcase-swift/LiferayScreens-Showcase-Swift/ViewControllers/Portlet/js/gallery.js [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+	var cards = document.getElementsByClassName('card');
+
+	for (let i = 0; i < cards.length; i++) {
+		cards[i].addEventListener('click', function(event) {
+			let card = cards[i];
+			let imgSrc = card.querySelector('img').src;
+			event.stopPropagation();
+
+
+								  if (isShow) {
```
Comment (nhpatt) [inline ios/Samples/Showcase-swift/LiferayScreens-Showcase-Swift/ViewControllers/Portlet/js/gallery.js]: space space space

File: ios/Samples/Showcase-swift/LiferayScreens-Showcase-Swift/ViewControllers/Portlet/js/gallery.js [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+								  isShow = false
+								  NetworkActivityIndicator.hide();
+								  }
+								  else {
+								  isShow = true;
+								  NetworkActivityIndicator.show();
+								  }
+
+		window.Screens.postMessage("gallery", imgSrc.slice(7));
```
Comment (nhpatt) [inline ios/Samples/Showcase-swift/LiferayScreens-Showcase-Swift/ViewControllers/Portlet/js/gallery.js]: let's mix "" y ''

Comment (victorg1991) [inline ios/Samples/Showcase-swift/LiferayScreens-Showcase-Swift/ViewControllers/Portlet/js/gallery.js]: Ups, missed that

File: ios/Framework/Core/Portlet/html/index.html (hunk 2 above)
Comment (victorg1991) [inline ios/Framework/Core/Portlet/html/index.html]: On iOS, yes :D

Comment (nhpatt) [inline ios/Framework/Core/Portlet/html/index.html]: then why var :P

Comment (victorg1991) [inline ios/Framework/Core/Portlet/html/index.html]: * se va silbando *

Comment (victorg1991) [inline ios/Framework/Core/Portlet/html/index.html]: I will change that :)

File: ios/Framework/Core/Portlet/js/Screens.js (hunk 4 above)
Comment (victorg1991) [inline ios/Framework/Core/Portlet/js/Screens.js]: done :D

**Comment by nhpatt** [comment]
👍

**Pull request merged**